pkgs/top-level/all-packages.nix

```diff
@@ -25,9 +25,18 @@ with pkgs;
   };
 
   # Used by wine, firefox with debugging version of Flash, ...
-  pkgsi686Linux = forceSystem "i686-linux" "i386";
+  pkgs-i686 =
+    if system == "x86_64-linux" then
+      forceSystem "i686-linux" "i386"
+    else
+      self;
+
+  callPackage-i686 = pkgs-i686.callPackage;
+
+  # DEPRECATED
+  pkgsi686Linux = pkgs-i686;
+  callPackage_i686 = callPackage-i686;
```

Comment:
I don't have a particular desire to do these renames.  I'm mostly worried about the packageOverrides fix.

I'm happy to remove the uses of the pkgs_i686 module argument, and add a trace warning for it as well, and we can leave it at that for now?